examples/message_passing/after.cpp

```diff
@@ -0,0 +1,57 @@
+// This example shows how to use caf::after
+
+#include "caf/all.hpp"
```

Comment:
The `all.hpp` header was added a long time ago when CAF was *a lot* smaller than it is today. I would no longer consider it best practice to use it. I already thought of deprecating it, but all of this becomes a non-issue if we finally do the switch to C++ modules.

Long story short: please include `caf_main`, `event_based_actor`, `actor_system`, etc.